oidc-controller/api/core/config.py

```diff
@@ -1,12 +1,98 @@
+import json
 import logging
+import sys
+import logging.config
+import structlog
 import os
 from enum import Enum
 from functools import lru_cache
 from typing import Optional, Union
 
 from pydantic import BaseSettings
 
-logger = logging.getLogger(__name__)
+from pathlib import Path
+
+logging.basicConfig(
+    format="%(message)s",
+    stream=sys.stdout,
+    level=logging.DEBUG,
+)
+
+# Use environment variable to determin logging format
+# fallback to logconf.json
+# finally default to true
+use_json_logs: bool
+
+time_stamp_format = "%Y-%m-%d %H:%M.%S"
```

Comment:
We should allow users to override this, and fall back on ISO timestamp by default (based on info [here](https://www.structlog.org/en/stable/standard-library.html) the default format would be `iso`).

Reply:
Same here, these settings should be exposed via ENVVARs and defaulted as mentioned above.